provider/core/src/hello_world.rs

```diff
@@ -80,31 +80,34 @@ pub struct HelloWorldProvider {
 }
 
 impl HelloWorldProvider {
+    pub const DATA: &'static [(icu_locid::Locale, &'static str)] = &[
```

Comment:
Nit: Maybe not `pub`?

Reply:
It's pub so the FS test can iterate over it